### PR TITLE
fix: default boilerplate path

### PR DESCRIPTION
**Pull request by yiyang5055**
add csi-release-tools to the program and do verify boilerplate does not work, because default boilerplate dir error.

### CODE REVIEW

**Comment by pohly** [comment]
/ok-to-test

Do you have a test PR in some other repo with this commit?

File: boilerplate/boilerplate.py [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
 
 # Rootdir defaults to the directory **above** the repo-infra dir.
-rootdir = os.path.dirname(__file__) + "./../../../"
+rootdir = os.path.dirname(__file__) + "./../../"
 rootdir = os.path.abspath(rootdir)
 parser.add_argument(
     "--rootdir", default=rootdir, help="root directory to examine")
 
-default_boilerplate_dir = os.path.join(rootdir, "csi-driver-nfs/hack/boilerplate")
+default_boilerplate_dir = os.path.join(rootdir, "hack/boilerplate")
```
Comment (pohly) [inline boilerplate/boilerplate.py:41]: This is broken. If `rootdir` is the directory above "release-tools", then there is no "hack/boilerplate" directory in it.

I think `os.path.abspath(os.path.dirname(__file__))` is the right default?

**Review by pohly — CHANGES_REQUESTED** [review]

File: boilerplate/boilerplate.py (hunk 1 above)
Comment (yiyang5055) [reply boilerplate/boilerplate.py:41]: sorry, `hack/boilerplate` is my project directory.
yes, `default_boilerplate_dir=os.path.abspath(os.path.dirname(__file__))` is right

**Comment by yiyang5055** [comment]
> /ok-to-test
> 
> Do you have a test PR in some other repo with this commit?

I tested it in my project after I modified it

**Comment by pohly** [comment]
I verified this change in csi-driver-host-path.

/lgtm
/approve

**Pull request merged**